File: Randoms/bubbleSort.c
#include <stdio.h>

void printArray(int arr[],int length){
for(int i=0;i<length;i++){
    printf("%d ",arr[i]);
}
}

int main(){
int arr[]={1,3,2,43,76,53,21,90,-21,-87,100};


int length=sizeof(arr)/sizeof(arr[0]);
printf("Original Array \n");
printArray(arr,length);
for(int i=0;i<length;i++){
    for(int j=0;j<length-1-i;j++){
        if(arr[j]>arr[j+1]){
            
            int temp=arr[j];
            arr[j]=arr[j+1];
            arr[j+1]=temp;
           
        }
    }
}
printf("\n  Array after sorting \n");
printArray(arr,length);

    return 0;
}

File: Randoms/BubbleSort.java
public class BubbleSort {

    static void bubble_sort(int[]arr){
        if(arr==null){
            System.out.println("Empty");
            return;
        }
       
        for (int i = 0; i < arr.length; i++) {
            
            for (int j = 0; j < arr.length-i; j++) {
                if(arr[j]>arr[j+1]){
                    //swap
            int temp=arr[j];
            arr[j]=arr[j+1];
            arr[j+1]=temp;
                }
            }
            
        }
        for (int i = 0; i < arr.length; i++) {
            System.out.print(" "+arr[i]);
        }
    }
public static void main(String[] args) {
        int[] arr={1,2,02,54,-87,-952,22,35,94,25,1};
        bubble_sort(arr);
    }
}
